cluster/addons/podsecuritypolicies-rbac/gce/event-exporter-binding.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: RoleBinding
+metadata:
+  name: system:podsecuritypolicy:event-exporter
+  namespace: kube-system
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+    kubernetes.io/cluster-service: "true"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: system:podsecuritypolicy:event-exporter
```

Comment:
logic behind sometimes using `system:podsecuritypolicy:...` as the role name and sometimes just `podsecuritypolicy:...`?

Reply:
The idea was to distinguish those policies that have a very specific use to support system pods from those that are more generally useful. Specifically, the privileged & restricted policies are intended for use outside the kube-system namespace.

Any suggestions for how to make this cleaner?

Reply:
typically, we've tried to namespace roles/bindings to the creating component so that there's no conflict with user-created roles (`system:...` for things created by the apiserver, `gce:...` for things created by gce deployment scripts, `kubeadm:...` for things created by kubeadm, etc)

Reply:
Done. Long-term we should be using namespaces, not prefixes for this (non-namespaced objects might require a different approach -- namespaces in name only?). For now I went with the `gce:` prefix. We may also want to consider fully-qualified names (I.e. k8s.io, cloud.google.com, etc.) at some point.